src/Language/PureScript/Docs/Types.hs

```diff
@@ -38,6 +37,22 @@ import Language.PureScript.Docs.RenderedCode as ReExports
 --------------------
 -- Types
 
+newtype PackageName = PackageName { getPackageName :: Text }
```

Comment:
So I think I'd rather define this type in its own module and not export the constructor.